src/index.html

```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="container">
```

Comment:
I believe you don't need this container
```suggestion
  <body>
```